docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -34,37 +32,31 @@ spec:
 < . . . >
 ----
 +
-For more information about node selection, see xref:about-node-selector_planning[About node selector].
+For more information about node selection, see link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors].
 +
 [NOTE]
 ====
 If your nodes do not have enough local disk space for temporary images, you can use a remote NFS location by setting the extra volumes feature ()`extraMounts`.
 ====
 Transport protocols::
-Due to the specifics of the storage transport protocols, some changes might be
-required for {OpenShiftShort}. Before making changes, review the following information:
+Some changes to the storage transport protocols might be required for {OpenShiftShort}. Before making changes, review the following information:
 +
-* Check the back-end sections in your `cinder.conf` file that are listed in the
-`enabled_backends` configuration option to determine the transport storage
-protocol that the back end uses.
+* If you use a `MachineConfig` to make changes to {OpenShiftShort} nodes, the nodes reboot.
+* Check the back-end sections that are listed in the `enabled_backends` configuration option in your `cinder.conf` file to determine the transport storage protocol that the back end uses.
 * Depending on the back end, you can find the transport protocol by viewing the `volume_driver` or `target_protocol` configuration options.
 * The `icssid` service, `multipathd` service, and `NVMe-TCP` kernel modules start automatically on data plane nodes.
-+
-[WARNING]
-If you use a `MachineConfig` to make changes to {OpenShiftShort} nodes, the nodes reboot.
 
-NFS::
-{OpenShiftShort} connects to NFS back ends without additional changes.
+NFS:::
+** {OpenShiftShort} connects to NFS back ends without additional changes.
 
-RBD/Ceph::
-{OpenShiftShort} connects to Ceph back ends without additional changes. You must provide credentials and configuration files to the services.
+Rados Block Device and {Ceph}:::
+** {OpenShiftShort} connects to {Ceph} back ends without additional changes. You must provide credentials and configuration files to the services.
```

Comment:
Why is it significant to mention that you must provide credentials and configuration files to the RBD and Ceph services? Is this different from other services?

Reply:
Yes, it is different to most of the other drivers.

In general you can pass your credentials (username, password, etc.) as configuration options, but with Ceph you actually require files to be there.